keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2703,7 +2704,13 @@ of this feature?** No.
 
 ## Implementation History
 
-(none yet)
+### Introduce v1beta2
```

Comment:
Since you're touching it, can you maybe just briefly add some history, e.g.:

- v1.19 - Alpha release
- v1.20 - graduated to Beta
- v1.21 - [any meaningful changes worth mentioning here?]
- v1.22 - initial support for width concept and watch initialiation

Probably for consistency with the above

- v1.23 - v1beta2 API (no changed comparing to v1beta1) [v1beta1 remain storage version]
- v1.24 - storage version changed to v1beta2

Reply:
done
I don't think there was any meaningful change in  `v1.21 `, maybe @MikeSpreitzer knows!